dev-libs/librevenge/librevenge-0.0.4.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="A helper library for REVerse ENGineered formats filters"
+DESCRIPTION="librevenge is a base library for writing document import \
+filters. It has interfaces for text documents, vector graphics, spreadsheets \
+and presentations."
+HOMEPAGE="http://sourceforge.net/p/libwpd/wiki/librevenge/"
+COPYRIGHT="Fridrich Strba
+	Laurent Alonso
+	David Tardon"
```

Comment:
I've grabbed the source archive and analyzed its ChangeLog to find the copyright years.
I also noticed that 2 authors are missing in the COPYRIGHT variable. The following table gives the number of commits each major contributor has authored. Remark: I've discarded 12 authors which have contributed with only 1 to 7 commits each.

Commits | Authors
-------- | --------
842 | Fridrich Strba
191 | David Tardon
167 | William Lachance
150 | Marc Maurer
78 | Laurent Alonso & osnola (=Alonso)

Using grep and friends I find this:
```
COPYRIGHT="2004-2015 Fridrich Strba
	2012-2016 David Tardon
	2002-2006, 2010 William Lachance
	2002-2006 Marc Maurer
	2011-2015 Laurent Alonso"
```